kong/tools/database_cache.lua

```diff
@@ -2,12 +2,19 @@ local utils = require "kong.tools.utils"
 local resty_lock = require "resty.lock"
 local json_encode = require("cjson.safe").encode
 local json_decode = require("cjson.safe").decode
+local lrucache = require "resty.lrucache"
```

Comment:
Depends on customer use. I'd expect it to be relatively high, based on "sessions", what I mean is that you'll most likely get multiple requests for a specific use. In that case the apis, cunsomers, credentials etc. will be the same for that session. Only when done the next session comes up.

but as said very speculative.